README.md

```diff
@@ -7,77 +7,69 @@ A Rust binding to OpenBSD's pledge(2) interface.
 
 ## Usage
 
-    #[macro_use] extern crate pledge;
-    use pledge::{pledge, Promise, ToPromiseString};
+    /* Rust 2015 only */ #[macro_use] extern crate pledge;
+    /* Rust 2018 only */ use pledge::{pledge, pledge_promises, pledge_execpromises};
 
     fn foo() {
         // make both promises and execpromises
-        match pledge![Stdio Proc Exec, Stdio Tty] {
-            Err(_) => println!("Failed to pledge"),
-            _ => ()
-        }
```

Comment:
there are three things I thought we can improve about this error handling

- it swallows the details, be it EFAULT/EINVAL/EPERM or one of our own
- it prints the error message to stdout when we would generally prefer stderr
- Option::unwrap or the try operator is more conventional for examples

Reply:
:+1: